packages/ui/src/ui/pages/system/Nodes/Nodes.js

```diff
@@ -39,17 +50,26 @@ class Nodes extends Component {
     };
 
     renderContent(theme) {
-        const {racks, rackGroups, containerWidth} = this.props;
-        const headingCN = block('elements-heading')({
+        const {racks, rackGroups, nodeType, containerWidth} = this.props;
+        const headingCN = cn('elements-heading')({
             size: 's',
             overview: 'yes',
         });
 
+        if (_.isEmpty(racks) && _.isEmpty(rackGroups)) {
+            return (
+                <NoContent
+                    warning={`There are no nodes for ${format.ReadableField(nodeType?.join(','))}`}
```

Comment:
Maybe change warning text to:
`There are no {nodeType}` because word 'nodes' duplicates

<img width="395" alt="image" src="https://github.com/ytsaurus/ytsaurus-ui/assets/67626299/363788af-8896-4309-abbd-41f8a85ad5df">

also it is better to add space after coma to `nodeType?.join(', ')`, because when 2 or more node types selected - warning text will be stuck together.
Now warning text might look like:
`There are no nodes for Chaos node,tablet nodes`